rpa-home/modules/ROOT/pages/glossary.adoc

```diff
@@ -7,24 +7,40 @@ The following glossary helps you understand MuleSoft terminology.
 == A
 
 ''''''
+
+=== Action Step
+
+The smallest components in a Workflow. Use action steps to build logical blocks and transactions in RPA Builder. 
+
 === Activity
 
-A basic unit of work in the Mule application flow. This can in turn be a connector that connects to an external system or a transform step that is used to transform data inside the application.
+A single unit of work in the automation project. An activity can be a bot task and its linked workflow, or a user task.
+
+=== Activity Library 
+
+Enables you to reuse certain activities that repeat during the automation of business processes, such as a login, to avoid creating the same workflow multiple times.
+
+=== Activity Parameter 
+
+Enables you to map variables that you can use not only in the Workflow, but also in the activities of the business process and gateways.
 
+=== Analysis Package 
+
+An error report that contains full information on the run in the same format as when you execute the Workflow locally.
```

Comment:
Not a 'report'. It's a *.zip file containing logfile of the workflow run and error screenshots. When opened / imported in RPA Builder it will display the info in the same format as when executed locally in Builder.